acceptance/gem_versions/.kitchen.yml

```diff
@@ -0,0 +1,47 @@
+driver:
+  name: ec2
+  instance_type: m3.medium
+  aws_ssh_key_id: serdar
```

Comment:
You could make these configurable by making them environment variables like `<%= ENV['AWS_SSH_KEY_ID'] %>`